server/chat/backend/agent/tools/introspection_tools.py

```
@@ -0,0 +1,856 @@
+
+
+# ---------------------------------------------------------------------------
+# Incidents
+# ---------------------------------------------------------------------------
+
+class ListIncidentsArgs(BaseModel):
+    status: Optional[str] = Field(
+        default=None,
+        description="Filter by incident status: investigating, analyzed, merged, or resolved.",
+    )
+    limit: int = Field(default=20, description="Max incidents to return (1–100, default 20).")
```

Ruff (0.15.17)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)